modules/shared/src/main/scala/io/constellationnetwork/schema/swap.scala

```diff
@@ -161,12 +161,17 @@ object swap {
   @derive(decoder, encoder, order, show)
   case class SpendTransaction(
     fee: SpendTransactionFee,
-    lastValidOrdinal: SnapshotOrdinal,
+    validUntilOrdinal: SnapshotOrdinal,
```

Comment:
"Until" means inclusive or exclusive? "lastValidOrdinal" wasn't ambiguous

Reply:
Notion doc isn't up to date for sure

Reply:
👍🏼